### PR TITLE
SASS-1689 - Update journey tests to include tests for the medical sec…

### PR DESCRIPTION
**Pull request by georgievh**
### Description
Re-orderes tests and includes new ones for the medical section.

Add a link to the relevant story in Jira
[SASS-1689](https://jira.tools.tax.service.gov.uk/browse/SASS-1689)

##### Before Reviewing
 - [x]  Have you pulled the branch down?
 - [x]  Have you assigned yourself to the PR? 
 - [x]  Have you moved the task to “in review” on JIRA? 
 - [x]  Have you checked to ensure all dependencies are up to date? 
 - [x] Have you checked to ensure its been rebased against the current version of main?

##### Whilst Reviewing
 - [x]  Have you run the journey tests?
 - [x]  Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?

##### After Reviewing
- [x] Have you checked for merge conflicts or any changes in the current main that may affect the current pull request? i.e. does it need another rebase?
 the current pull request?
 - [x]  Have you checked to make sure there are no builds in the pipeline before you merge? 
 - [x]  Have you moved the task to “in pipeline” on Jira?

##### Before creating PR
 - [x]  Have you run the journey tests? 
 - [x]  Have you addressed warnings where appropriate?
 - [x]  Have you rebased against the current version of main?

### CODE REVIEW

File: src/test/resources/features/04_employment.feature [hunk 1]
```diff
@@ -1447,14 +1099,14 @@ Feature: Employment Journeys - Income Tax Submission
...
     When the user logs into the service with the following details
-      |Redirect url       |/2021/start                                          |
-      |Credential Strength|strong                                               |
-      |Confidence Level   |200                                                  |
-      |Affinity Group     |Individual                                           |
-      |Nino               |BB444444A                                            |
-      |Enrolment Key 1    |HMRC-MTD-IT                                          |
-      |Identifier Name 1  |MTDITID                                              |
-      |Identifier Value 1 |1234567890                                           |
+      | Redirect url        | /2021/start |
```
Comment (joeyymoore) [inline src/test/resources/features/04_employment.feature:1102]: This isn't something you've done, but on line 1120 i don't understand why it changes travel section to no, this is all about accommodation being no so i think lines 1120-1124 can be removed

File: src/test/resources/features/04_employment.feature [hunk 2]
```diff
@@ -1503,14 +1155,14 @@ Feature: Employment Journeys - Income Tax Submission
...
```
Comment (joeyymoore) [inline src/test/resources/features/04_employment.feature:1158]: Same as above comment, not sure why someones changing utilities to no aswell as travel. Lines 1172-1180 can be removed.

**Review by joeyymoore — CHANGES_REQUESTED** [review]

File: src/test/resources/features/04_employment.feature (hunk 1 above)
Comment (georgievh) [reply src/test/resources/features/04_employment.feature:1102]: Removed as suggested

**Review by joeyymoore — APPROVED** [review]

File: src/test/resources/features/04_employment.feature (hunk 2 above)
Comment (georgievh) [reply src/test/resources/features/04_employment.feature:1152]: Removed as suggested.

**Comment by joeyymoore** [comment]
Will merge alongside the Dewit changes on employment-fe for https://jira.tools.tax.service.gov.uk/browse/SASS-1646

**Pull request merged**